.gitattributes

```diff
@@ -0,0 +1,7 @@
+# Auto detect text files and perform LF normalization
+# https://www.davidlaing.com/2012/09/19/customise-your-gitattributes-to-become-a-git-ninja/
+* text=auto
+
+# Linguist override
+*.re linguist-language=Reason
+*.rei linguist-language=Reason
```

Comment:
I think this part is now unecessary. A few week GitHub added auto detection of Reason language.

Not sure about adding a file for the text=auto thing. I was about to delete all .gitattributes from all repos of this org.

What do you think?

Reply:
No, Reason is still misdetected. The change is just to remove Reason from the OCaml group, and the root cause of the problem is that GitHub still supports the rarely used re extension for C++.